docs/QSG/setup_rhel.md

```diff
@@ -310,11 +309,16 @@ Examples are available on [github](https://github.com/daos-stack/daos/tree/relea
 			log_file: /tmp/daos_engine_0.log
 			env_vars:
 				- CRT_TIMEOUT=30
-			scm_mount: /mnt/daos0
-			scm_class: dcpm
-			scm_list: [/dev/pmem0]
-			bdev_class: nvme
-			bdev_list: ["0000:81:00.0"]  # generate regular nvme.conf
+			storage:
+			-
+				class: dcpm
+				scm_mount: /tmp/pmem0
```

Comment:
should this scm_mount value be /mnt/daos0 instead?

Reply:
thanks for catching - fixed